src/scenario/action_types/send_standard_message.c

```diff
@@ -0,0 +1,13 @@
+#include "send_standard_message.h"
+
+#include "city/message.h"
+#include "empire/trade_prices.h"
+
+int scenario_action_type_send_standard_message_execute(scenario_action_t *action)
+{
+    int32_t text_id = action->parameter1;
+
+    city_message_post(1, text_id, 0, 0);
```

Comment:
You should check if the `text_id` is valid to prevent crashes. Remember: never trust user input.